### PR TITLE
If error

### PR DESCRIPTION
**Pull request by yamaguchi1024**

### CODE REVIEW

File: inc/boost/module.modulemap [hunk 1]
```diff
@@ -2324,18 +2324,26 @@ module boost_graph_pending_random_wrapper {
...
     module "pending__property_serialize" { header "pending/property_serialize.hpp" export * }
     module "pending__queue" { header "pending/queue.hpp" export * }
-    module "pending__relaxed_heap" { header "pending/relaxed_heap.hpp" export * }
     module "pending__stringtok" { header "pending/stringtok.hpp" export * }
+  /*
+  All these modules caused this "declaration of 'if_'.." error, so removed them from modulemap
+/home/travis/build/Teemperor/boost-compile/inc/boost/property_map/property_map.hpp:60:17: error:
+      declaration of 'if_' must be imported from module 'boost_mpl.mpl__bind'
+      before it is required
+    boost::mpl::if_<is_property_map<PA>,
```
Comment (vgvassilev) [inline inc/boost/module.modulemap:2340]: Would that mean that we are missing a `#include <mpl/bind>`? Maybe it is worth in submitting a PR with adding this header file.

Comment (Teemperor) [reply inc/boost/module.modulemap:2340]: This quick check doesn't seem to confirm this, so it's a modules bug:
```diff
diff --git a/inc/boost/property_map/property_map.hpp b/inc/boost/property_map/property_map.hpp
index cd6dd55..b822602 100644
--- a/inc/boost/property_map/property_map.hpp
+++ b/inc/boost/property_map/property_map.hpp
@@ -55,6 +55,9 @@ namespace boost {
  
   struct null_property_traits {};
  
+  #ifndef BOOST_MPL_IF_HPP_INCLUDED
+   #error "Foo"
+  #endif
   template <typename PA>
   struct property_traits :
     boost::mpl::if_<is_property_map<PA>,
```

But directly including the header from this file fixes it, so somehow this decl isn't propagated when implicitly included by something else.

Comment (Teemperor) [reply inc/boost/module.modulemap:2340]: Ok, figured it out.

**Comment by Teemperor** [comment]
Properly fixed in #64

**Pull request closed without merging**